gallery/src/components/NxButton/NxButtonIconOnlyExample.tsx

```diff
@@ -11,9 +11,12 @@ import { NxButton, NxFontAwesomeIcon } from '@sonatype/react-shared-components';
 
 const NxButtonIconOnlyExample = () =>
   <div className="nx-btn-bar">
-    <NxButton variant="icon-only"><NxFontAwesomeIcon icon={faExclamationTriangle}/></NxButton>
-    <NxButton variant="icon-only"><NxFontAwesomeIcon icon={faEdit}/></NxButton>
-    <NxButton variant="icon-only"><NxFontAwesomeIcon icon={faSave}/></NxButton>
+    <NxButton variant="icon-only"><NxFontAwesomeIcon aria-hidden={false}
+                                                     aria-label='alert'
+                                                     icon={faExclamationTriangle}/>
+    </NxButton>
+    <NxButton variant="icon-only"><NxFontAwesomeIcon aria-hidden={false} aria-label='edit' icon={faEdit}/></NxButton>
+    <NxButton variant="icon-only"><NxFontAwesomeIcon aria-hidden={false} aria-label='save' icon={faSave}/></NxButton>
```

Comment:
When adding labels, I capitalized the text. Does it matter?

Reply:
I don't think it matters, though I had noticed a lot of aria-label examples used lower-case text.

Reply:
It seems it doesn't matter, it will be read the same.